dvc/remote/local/__init__.py

```diff
@@ -136,29 +122,44 @@ def link(self, from_info, to_info):
         self._link(from_info, to_info, self.cache_types)
 
     def _link(self, from_info, to_info, link_types):
```

Comment:
I guess you meant to remove this one, as you have it in RemoteBASE already, right?